docs/getting-started.md

```diff
@@ -0,0 +1,70 @@
+# Getting started
+
+## Prerequisites
+
+You'll need the following tools installed in order to run the project locally:
+
+-   [Composer](https://getcomposer.org/download/) v1.3.0+
+-   MySQL v5.7+ or PostgreSQL 9.5+
+
+Once you have the prerequisites installed on your machine, and have the project checked out locally, run these commands in your terminal of choice:
+
+## Install dependencies
+
+From the root of the project run:
+
+```shell script
+composer install
+```
+
+## Create a Database
+
+Next up, you need to create a database for the project. Both MySQL 5.5+ and PostgreSQL 9.5+ are supported.
```

Comment:
Wonder if we should only specify MySQL as that's what we use in production?

Reply:
I have changed it to only mention MySQL to be sure. I have also formatted the link syntax and added spacing to the headers. Thanks!